examples/ci_cd/gitlab-ci-cd/main.tf

```diff
@@ -61,29 +61,20 @@ data "aws_eks_cluster_auth" "cluster" {
 }
 
 locals {
-  tenant      = "aws001"  # AWS account name or unique id for tenant
-  environment = "preprod" # Environment area eg., preprod or prod
-  zone        = "qa"      # Environment with in one sub_tenant or business unit
-
-  kubernetes_version = "1.21"
-
-  vpc_cidr     = "10.2.0.0/16"
-  vpc_name     = join("-", [local.tenant, local.environment, local.zone, "vpc"])
-  cluster_name = join("-", [local.tenant, local.environment, local.zone, "eks"])
-
-  terraform_version = "Terraform v1.0.1"
+  vpc_name     = join("-", [var.tenant, var.environment, var.zone, "vpc"])
+  cluster_name = join("-", [var.tenant, var.environment, var.zone, "eks"])
 }
 
 module "aws_vpc" {
-  source  = "terraform-aws-modules/vpc/aws"
-  version = "v3.2.0"
+  source = "terraform-aws-modules/vpc/aws"
+  //version = "v3.2.0"
```

Comment:
Can we uncomment this? I accidentally removed it before, but I understand now it's used for recording the module version.

Reply:
Done! updated